src/RequestHandler/outofprocess/serverprocess.cxx

```diff
@@ -1039,55 +1014,26 @@ SERVER_PROCESS::StartProcess(
         }
     }
 
-//    if (FAILED(hr))
-//    {
-//        if (strEventMsg.IsEmpty())
-//        {
-//            if (!fDonePrepareCommandLine)
-//            {
-//                strEventMsg.SafeSnwprintf(
-//                    m_struAppFullPath.QueryStr(),
-//                    ASPNETCORE_EVENT_PROCESS_START_INTERNAL_ERROR_MSG,
-//                    hr);
-//            }
-//            else
-//            {
-//                strEventMsg.SafeSnwprintf(
-//                    ASPNETCORE_EVENT_PROCESS_START_POSTCREATE_ERROR_MSG,
-//                    m_struAppFullPath.QueryStr(),
-//                    m_pszRootApplicationPath.QueryStr(),
-//                    struCommandLine.QueryStr(),
-//                    hr);
-//            }
-//        }
-//
-//        apsz[0] = strEventMsg.QueryStr();
-//
-//        // not checking return code because if ReportEvent
-//        // fails, we cannot do anything.
-//        //
-//        if (FORWARDING_HANDLER::QueryEventLog() != NULL)
-//        {
-//            ReportEventW(FORWARDING_HANDLER::QueryEventLog(),
-//                EVENTLOG_ERROR_TYPE,
-//                0,
-//                ASPNETCORE_EVENT_PROCESS_START_ERROR,
-//                NULL,
-//                1,
-//                0,
-//                apsz,
-//                NULL);
-//        }
-//
-//        // FREB log
-//        if (ANCMEvents::ANCM_START_APPLICATION_FAIL::IsEnabled(context->GetTraceContext()))
-//        {
-//            ANCMEvents::ANCM_START_APPLICATION_FAIL::RaiseEvent(
-//                context->GetTraceContext(),
-//                NULL,
-//                strEventMsg.QueryStr());
-//        }
-//    }
+    if (FAILED(hr))
+    {
+        if (strEventMsg.IsEmpty())
+        {
+              strEventMsg.SafeSnwprintf(
```

Comment:
This doesn't mirror the error message that were there before. Do we still want to check 
```cpp
if (!fDonePrepareCommandLine)
{
    strEventMsg.SafeSnwprintf(
        m_struAppFullPath.QueryStr(),
        ASPNETCORE_EVENT_PROCESS_START_INTERNAL_ERROR_MSG,
        hr);
}
else 
{
...
```